elements/em.html

```diff
@@ -0,0 +1,44 @@
+<article class="article">
+  <header class="article-header">
+    <h2>
+      <code>
+        &lt;em&gt;
+      </code>
+    </h2>
+    <div class="base-words">
+      <div class="base-word-line">
+        English Base: <strong>Emphasis</strong>
+      </div>
+      <div class="base-word-line">
+        Spanish translation/Traducción en Español: <strong></strong>
+      </div>
+    </div>
+  </header>
+  <div class="definition">
+    <div class="definition-box definition-spanish">
+      <h4>Definici&oacute;n</h4>
+      <p></p>
+    </div>
+    <div class="definition-box definition-english">
+      <h4>Definition</h4>
+      <p>Element used to stress emphasis or show priority or force on a particular section of text so it stands out from its surrounding text.  Emphasis is a logical state meaning it uses physical state changes to change the text.  As it's logical, it can be one physical change or multiple.  Emphasis usually changes the selected text to italics.  It does not need to be the same for every website.  Example:  You could set up emphasis for a document on quitting smoking to always assign multiple physical changes on the words "Stop Smoking Now!" where it is in italics, font-color of royal blue and always underlined.</p>
```

Comment:
I think this is a bit technical for complete HTML newbies: "Emphasis is a logical state meaning it uses physical state changes to change the text.  As it's logical, it can be one physical change or multiple."

The next sentence "Emphasis usually changes the selected text to italics." could add some details about why only usually like: "Most browsers display text in an emphasis tag as italic, but you can override that.  For example," ...